index.ts

```diff
@@ -80,13 +80,13 @@ async function registerOnOrigins({
 	injectOnExistingTabs(newOrigins || [], manifest);
 }
 
-(async () => {
+chrome.runtime.onStartup.addListener(async () => {
```

Comment:
Wow, looks like a very simple fix. Great job! 👏👏👏

Reply:
Unfortunately… it's huge issue 😅 

I have good news and bad news.

Good: I think this was already fixed in the previous update of this module: https://github.com/fregante/webext-dynamic-content-scripts/pull/30

Bad (for me): It had only fixed it for github.com

This change maybe fixes the second part, but potentially breaks this module altogether. I'm waiting for Chrome to trigger the bug so I can test it.

In the meanwhile I'll push an update to Refined GitHub to fix it for github.com